test-it/ViaTests/Log10.via

```diff
@@ -0,0 +1,1320 @@
+// Autogenerated Vireo assembly file
+// Generated on 03/07/2019 15:38:14
+//---------------------------------------------------
+//VI Definition: ::Web Server::Interactive::WebApp::index.gviweb
```

Comment:
Thanks for adding extensive test cases. This is great. If the VIA files are generated from GVIs (and you are planning to check in the VIs in G test suite on ASW side), I would recommend keeping the names of the VIA files same as .GVI files (TestVI*.via).

Reply:
I was waiting for the outcome of test location meeting with GregR before responding. Since TestVI prefix was not considered as a recommendation, I feel we should not adopt it. Also naming conventions are brittle.

Carlos mentioned creating nuget packages from NextGen containing via files for consumption by Vireo. I feel we should look more into it as an option.